packages/rancher-logging/package.yaml

```diff
@@ -1,6 +1,6 @@
 url: https://kubernetes-charts.banzaicloud.com/charts/logging-operator-3.9.0.tgz
 packageVersion: 00
-releaseCandidateVersion: 08
+releaseCandidateVersion: 09
```

Comment:
Once rest of PR is confirmed you'll need to pull in upstream again and bump packageVersion.

Reply:
Bumped the packageVersion, does the releaseCandidateVersion stay the same or get reset?

Reply:
Needs to get reset to 01

Reply:
done